test/rubygems/test_gem_installer.rb

```diff
@@ -1142,14 +1142,24 @@ def test_install_extension_dir_is_removed_on_reinstall
   end
 
   def test_find_lib_file_after_install
```

Comment:
ah, I understand to your means. Thanks for your explanation. I'm going to add your original tests under ruby 1.9.3.